### PR TITLE
Releasing Neo4j 4.2.15 and 4.4.4

### PR DESCRIPTION
**Pull request by jennyowen**
Thanks!

### CODE REVIEW

**Comment by yosifkit** [comment]
It seems there was some sort of templating issue that also copied the template Dockerfile into the `local-package/` directory. Small and harmless when copied into the image, but does conflate the diff a bit.

**Comment by jennyowen** [comment]
Gah! I suppose this is what happens when you try to refactor old complicated Makefiles :cry: 

I've fixed it, and there was another Neo4j release overnight, so that's included in this PR too now.

**Comment by yosifkit** [comment]
It looks like the files from the `COPY ./local-package/* /startup/` are no longer used by the `RUN` line, so it should be moved later in the Dockerfile to optimize `docker build` cache.

**Comment by jennyowen** [comment]
Hi @yosifkit you're right that the COPY could be moved to after the RUN *in the Dockerfile we publish* but it would break our build process for when Neo4j is under development.

We only upload a Neo4j to `https://dist.neo4j.org/neo4j-<whatever>-unix.tar.gz` once a new version has been developed, tested and released. Until then, we make the docker image by putting neo4j in `./local-package/` and setting the `NEO4J_URI` build arg to point to the copied file when we do `docker build`.

If it's important to you that we keep the build cache as optimised as possible, I could still make the changes you request to just the published Dockerfiles. Although I'm hesitant to make them too different to the ones we test.

So TLDR: I would prefer not do that unless it's important for compliance with the docker official images library.

**Comment by jennyowen** [comment]
Thank you both! :heart:
